Formula/asc.rb

```diff
@@ -0,0 +1,22 @@
+class Asc < Formula
+  desc "App Store Connect API access via Swift command-line tool"
+  homepage "https://github.com/Blackjacx/Assist"
+  url "https://github.com/Blackjacx/Assist.git", tag: "0.1.1", revision: "6f07c2e97eb72bf2250b5bca888604007951c081"
+  license "MIT"
+  head "https://github.com/Blackjacx/Assist.git", branch: "develop"
+
+  depends_on xcode: [">13.1", :build]
+  depends_on macos: [
+    :catalina,
+    :big_sur,
+    :monterey,
+  ]
```

Comment:
```suggestion
  depends_on macos: :catalina
```
I don't think this takes an `Array` value. This is what you need to specify Catalina or newer, though.

This is a bit redundant, however, since this is already implied by your Xcode requirement, which is even stricter.